monkeys/drip-drop/values-drip-drop.yaml

```diff
@@ -0,0 +1,6 @@
+name: "drip-drop"
+replicas: 3
+env:
+    sleep: "1"
```

Comment:
Let's make the sleep be `0.001` so we basically instantly re-establish the connection.